kolibri/plugins/management/assets/src/views/user-page/user-create-modal.vue

```diff
@@ -119,61 +117,70 @@
       LEARNER: () => UserKinds.LEARNER,
       COACH: () => UserKinds.COACH,
       ADMIN: () => UserKinds.ADMIN,
-      statusMessage() {
-        if (this.errorMessage) {
-          return this.errorMessage;
-        } else if (this.confirmationMessage) {
-          return this.confirmationMessage;
+      usernameAlreadyExists() {
+        return this.users.find(user => user.username === this.username);
```

Comment:
I know I'm not supposed to be worrying about efficiency

...

...

but this is going to be going through a potentially large list of users on every single keypress. I want to suggest using `v-model.lazy`, but I remember trying it at some point and it not behaving as expected.